File: ruby/p7.rb
#!/usr/bin/ruby


def isPrime(num)
	i = 2
	numroot = Math.sqrt(num)
	while i <= numroot do
		if num % i == 0
			return false
		end
		i+=1
	end
	return true
end


i=0
cnt = 2
while i < 10001 do
	if isPrime(cnt)
		i+=1
		puts "#{i}, #{cnt}\n"
	end
	cnt+=1	
end

File: python/p7.py
#!/usr/bin/python

def isPrime(num):
	i=2
	while i*i <= num:
		if num%i==0:
			return False
		i+=1
	return True	


i=0
cnt=2
while i < 10001:
	if isPrime(cnt):
		i+=1
		print i,cnt
	cnt+=1	

